src/components/Side.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Side() {
```

Comment:
Lets rename this  component to something more descriptive like TagSelector.

Reply:
Done!